coreos-ostree-importer/send-ostree-import-request.py

```diff
@@ -90,7 +90,6 @@ def get_bucket_and_prefix(path):
 
 def validate_response(response):
     if response["status"].lower() == "failure":
-        # https://pagure.io/robosignatory/pull-request/38
         if "failure-message" not in response:
             raise Exception("Signing failed")
         raise Exception(f"Signing failed: {response['failure-message']}")
```

Comment:
Seems like these should be updated too to something like `Importing failed` ?

Reply:
+1 - added a new commit with the fix